.github/dependabot.yml

```diff
@@ -1,5 +1,4 @@
 ---
-
```

Comment:
There is no reason to remove this blank line that comes from the skeleton.

Reply:
@jsf9k resolved with [04b5d49](https://github.com/cisagov/con-pca-api/pull/637/commits/04b5d4920cb3ec0007f909063c3f407a92b660d1)